middleware/packet-forward-middleware/Makefile

```diff
@@ -147,6 +147,7 @@ mocks: $(MOCKS_DIR)
 	mockgen -package=mock -destination=./test/mock/bank_keeper.go $(GOMOD)/router/types BankKeeper
 	mockgen -package=mock -destination=./test/mock/ics4_wrapper.go github.com/cosmos/ibc-go/v7/modules/core/05-port/types ICS4Wrapper
 	mockgen -package=mock -destination=./test/mock/ibc_module.go github.com/cosmos/ibc-go/v7/modules/core/05-port/types IBCModule
+	mockgen -package=mock -destination=./test/mock/channel_keeper.go $(GOMOD)/router/types ChannelKeeper
```

Comment:
Maybe add  `go install go.uber.org/mock/mockgen@v0.2.0` into the `mocks` target, or a comment to use this for mockgen now.

Reply:
good idea, i'll add this now!